yalla-coop/working-class-history
# Add articles

#### Issues this PR relates to
#79

**how to test:** 
  1- install the excel file from issue #79.
  2- convert it to CSV file using any online tool to convert excel to CSV (suggestion https://www.zamzar.com/).
  3- open the project and `npm i`. to install react parser library.
  4- go to 'http://localhost:3000/add-new-articles' and upload the CSV file (you can see the articles positing on network tab on dev tools after clicking post articles button).

note : this will produce side effect which is creating new rows on the baserow database "articles table". 255 new row and because there is no query to remove rows on baserow I had to remove it manually (it takes about 10min) click right and delete row on each new row that results from the test.

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs
